### PR TITLE
Rename: Scope State Processor

### PR DESCRIPTION
**Pull request by bruno-garcia**

### CODE REVIEW

File: samples/Sentry.Samples.Console.Customized/Program.cs [hunk 1]
```diff
@@ -107,6 +107,9 @@ private static async Task Main(string[] args)
             {
                 client.DefaultRequestHeaders.TryAddWithoutValidation("CustomHeader", new[] { "my value" });
             };
+
+            // Control/override how to apple the State object into the scope
```
Comment (Tyrrrz) [inline samples/Sentry.Samples.Console.Customized/Program.cs:111]: apple -> a pply

**Review by Tyrrrz — APPROVED** [review]

File: samples/Sentry.Samples.Console.Customized/Program.cs (hunk 1 above)
Comment (bruno-garcia) [reply samples/Sentry.Samples.Console.Customized/Program.cs:111]: Thanks!

Comment (bruno-garcia) [inline samples/Sentry.Samples.Console.Customized/Program.cs:111]:
```suggestion
            // Control/override how to apply the State object into the scope
```

**Comment by bruno-garcia** [comment]
I'll merge in a bit. Just sent a diff that includes this branch as upstream so I don't want to break that link.

We'll validate this hook, then ship it if it works for them.

**Comment by Tyrrrz** [comment]
I also created a (broken) PR to merge the backport branch to current main:
https://github.com/getsentry/sentry-dotnet/pull/597

**Comment by bruno-garcia** [comment]
I disabled auto delete of head branches

**Pull request merged**